docs/tutorials/get-started.md

```diff
@@ -146,21 +106,23 @@ paste the key there.)
 
 In this example we are using a simple example of a webservice and
 change its configuration to use a different message. The easiest
-way is to edit your fork of `flux-get-started` and change the
-`PODINFO_UI_MESSAGE` env var to `Welcome to Flux`.
+way is to edit your fork of `flux-get-started` and change the `PODINFO_UI_COLOR` env var to `blue`.
 
 Replace `YOURUSER` in
 `https://github.com/YOURUSER/flux-get-started/blob/master/workloads/podinfo-dep.yaml`
 with your GitHub ID), open the URL in your browser, edit the file,
 change the env var value and commit the file.
 
-By default, Flux git pull frequency is set to 5 minutes.
-You can tell Flux to sync the changes immediately with:
+You can check out the Flux logs with:
 
 ```sh
-fluxctl --k8s-fwd-ns=fluxcd sync
+kubectl -n default logs deployment/flux -f
 ```
 
+The default sync frequency is 5 minutes. This can be tweaked easily.
```

Comment:
Not sure if this was already there or is new, but my thought while reading this was "can be tweaked easily how?"

Reply:
This was already there. Let's address it in https://github.com/fluxcd/flux/pull/2298